rubocop-on-rbs.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |spec|
   spec.metadata['homepage_uri'] = spec.homepage
   spec.metadata['source_code_uri'] = spec.homepage
   spec.metadata['changelog_uri'] = spec.homepage
+  spec.metadata['default_lint_roller_plugin'] = 'RuboCop::RBS::Plugin'
```

Comment:
Why need this change?

Reply:
Required for the following:
https://github.com/standardrb/lint_roller?tab=readme-ov-file#packaging-a-plugin-in-a-gem